src/Sylius/Bundle/ShopBundle/Resources/config/app/config.yml

```diff
@@ -4,24 +4,29 @@
 imports:
     - { resource: "@SyliusUiBundle/Resources/config/app/config.yml" }
 
+    - { resource: 'twig_hooks/**/*.yaml' }
     - { resource: "@SyliusShopBundle/Resources/config/app/sylius/sylius_mailer.yml" }
     - { resource: "@SyliusShopBundle/Resources/config/grids/account/order.yml" }
     - { resource: "@SyliusShopBundle/Resources/config/grids/product.yml" }
 
 parameters:
     sylius.security.shop_regex: "^/(?!%sylius_admin.path_name%|api/.*|api$|media/.*)[^/]++"
 
+twig:
+    paths:
+        '%kernel.project_dir%/src/Sylius/Bundle/ShopBundle/Resources/assets/': 'SyliusShopAssets'
```

Comment:
I have a tingling feeling this will fail on end apps.

Reply:
Yes it does. I already raised this question on slack but no answer how this will be solved. As I did test installing Sylius 2.0 as a vendor dependency and this is 100% an issue. For now I just added a composer patch in my local testing environment to change this to `'%kernel.project_dir%/vendor/sylius/sylius/src/Sylius/Bundle/ShopBundle/Resources/assets/':`
However, it should be addressed while Sylius/Sylius-Standard will be adapted to work with Sylius 2.0